docs/verifiable_credentials_examples/talent_protocol_passports/panagiotis_matsinopoulos.json

```diff
@@ -0,0 +1,63 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "https://passport.talentprotocol.com/passports/0x324e9E13dd19528D0F390201923d17c4B7E94462",
+  "type": [
+    "VerifiableCredential",
+    "TalentProtocolPassportCredential"
+  ],
+  "name": "Talent Protocol Credentials",
+  "description": "A verification of Talent Protocol Credentials issued by Talent Protocol to Builders",
+  "issuer": {
+    "id": "https://passport.talentprotocol.com/credentials/issuers/talentprotocol.eth",
```

Comment:
we own the talentprotocol.eth so we can create the verifications.talentprotocol.eth domain and use that here, what do you think?

Reply:
sounds good to me.